### ISSUE TITLE
Some confusion

### ISSUE THREAD

**Issue by hongliang5316** [openresty/lemplate#8]
Hi, @agentzh 
When i use `/opt/lemplate/lemplate --compile template/*.tt2 > templates.lua`
it occurs  `Name "Text::Glob::seperator" used only once: possible typo at Text/Glob.pm line 24.` and
the `templates.lua` is generated successful.

**Comment by agentzh** [comment]
@hongliang5316 That warning can be safely ignored. I've never seen that thing myself. Maybe your Text::Glob module or Lemplate are too old?

**Comment by ingydotnet** [comment]
fwiw, I see that warning every time too, and I'm pretty sure I have the latest modules installed.

**Comment by agentzh** [comment]
@ingydotnet Since I cannot reproduce it myself, will you dig it up? Thanks!

**Comment by ingydotnet** [comment]
I can reproduce this with in the `openresty/lemplate` repo clone dir like this:
```
$ ./lemplate >/dev/null
Name "Text::Glob::seperator" used only once: possible typo at Text/Glob.pm line 24.
```

**Comment by ingydotnet** [comment]
I'll look for a quick fix.

**Comment by ingydotnet** [comment]
Fixed in #9.

**Comment by agentzh** [comment]
Already merged @ingydotnet's workaround and released as CPAN version 0.15.
